src/components/CarouselDishes/CarouselDishes.jsx

```diff
@@ -1,64 +1,15 @@
 import { AppCarousel } from '@/shared/AppCarousel/AppCarousel';
 import DishCard from '../DishCard/DishCard';
+import { CarouselDishesPropTypes } from './CarouselDishes.props';
 
-// Mok array of disches
-const dishArray = [
-  {
-    dishname: 'Spaghetti',
-    price: 12.99,
-    description: 'Classic Italian pasta with savory Bolognese sauce.',
-    image: 'https://via.placeholder.com/300',
-  },
-  {
-    dishname: 'Grilled Salmon',
-    price: 18.99,
-    description: 'Delicious grilled salmon seasoned to perfection.',
-    image: 'https://via.placeholder.com/300',
-  },
-  {
-    dishname: 'Vegetarian Pizza',
-    price: 15.99,
-    description: 'A delightful pizza with a variety of fresh vegetables.',
-    image: 'https://via.placeholder.com/300',
-  },
-  {
-    dishname: 'Chicken Alfredo',
-    price: 14.99,
-    description: 'Creamy Alfredo sauce with tender chicken over pasta.',
-    image: 'https://via.placeholder.com/300',
-  },
-  {
-    dishname: 'Caesar Salad',
-    price: 9.99,
-    description: 'Fresh and crisp Caesar salad with homemade dressing.',
-    image: 'https://via.placeholder.com/300',
-  },
-  {
-    dishname: 'Beef Tacos',
-    price: 11.99,
-    description: 'Savory beef tacos with a variety of toppings.',
-    image: 'https://via.placeholder.com/300',
-  },
-  {
-    dishname: 'Chocolate Brownie',
-    price: 7.99,
-    description: 'Decadent chocolate brownie with a rich fudge topping.',
-    image: 'https://via.placeholder.com/300',
-  },
-  {
-    dishname: 'Fruit Smoothie',
-    price: 6.99,
-    description: 'Refreshing fruit smoothie made with a blend of fresh fruits.',
-    image: 'https://via.placeholder.com/300',
-  },
-];
-
-export const CarouselDishes = () => {
+export const CarouselDishes = ({ data }) => {
   return (
     <AppCarousel>
-      {dishArray.map((data, index) => (
-        <DishCard key={index} dishInfo={data} />
+      {data.map((item, index) => (
+        <DishCard key={index} dishInfo={item} />
```

Comment:
Interested to hear your opinion on the unique data that should be passed to the `key` props when mapping items. Here are a few articles I've come across recently:

https://legacy.reactjs.org/docs/lists-and-keys.html#:~:text=We%20don't%20recommend%20using,cause%20issues%20with%20component%20state.

https://medium.com/@vraa/why-using-an-index-as-key-in-react-is-probably-a-bad-idea-7543de68b17c

Reply:
‘key={I’d}’, index not good practice.